experimenter/experimenter/experiments/models.py

```diff
@@ -439,10 +439,19 @@ def start_date(self):
                 self.save()
                 return self._start_date
 
+    @property
+    def release_date(self):
+        if self.is_first_run:
+            return self.proposed_release_date
+
+    @property
+    def enrollment_start_date(self):
+        return self.release_date or self.start_date
+
```

Comment:
These two fields replace the need for a helper function, and make it clear that:
1. `self.proposed_release_date` is dependent on `is_first_run`
2. The start date for enrollment will either be `release_date` or `start_date` in all places